src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -398,39 +398,41 @@ public void resetComments(PullRequestRef pr,
       StashUser sonarUser,
       StashClient stashClient) {
     try {
-      // Let's call this "diffRep_loop"
-      for (StashComment comment : diffReport.getComments()) {
-
-        // delete comment only if published by the current SQ user
-        if (sonarUser.getId() != comment.getAuthor().getId()) {
-          continue;
-          // Next element in "diffRep_loop"
-
-          // comment contains tasks which cannot be deleted => do nothing
-        } else if (comment.containsPermanentTasks()) {
-          LOGGER.debug("Comment \"{}\" (path:\"{}\", line:\"{}\")"
-                  + "CANNOT be deleted because one of its tasks is not deletable.", comment.getId(),
-              comment.getPath(),
-              comment.getLine());
-          continue;  // Next element in "diffRep_loop"
-        }
-
-        // delete tasks linked to the current comment
-        for (StashTask task : comment.getTasks()) {
-          stashClient.deleteTaskOnComment(task);
-        }
-
-        stashClient.deletePullRequestComment(pr, comment);
-      }
+      // FIXME delete tasks on file-wide comments
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)